drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/loaders/project_unit_tracker.rb

```diff
@@ -74,7 +75,8 @@ def assign_specific_unit(enrollment_pk:, unit_id:, fallback_start_date: nil)
 
     # gives enrollments with same household id the same unit
```

Comment:
Oh whoops, I totally missed this. I added a separate thing to check for households. I will remove it.